libs/services/designer-client-services/src/lib/base/gateway.ts

```diff
@@ -29,7 +29,7 @@ export class BaseGatewayService implements IGatewayService {
   }
 
   public getGateways(subscriptionId: string, connectorName: string): Promise<Gateway[]> {
-    if (!subscriptionId || !connectorName) return Promise.resolve([]);
+    if (!connectorName) return Promise.resolve([]);
```

Comment:
This might need a config check here.

Something like:

```ts
const isSubscriptionRequired = !this.getConfig?.().disableSubscriptionLookup;
if ((isSubscriptionRequired && !subscriptionId) || !connectorName) {
  return Promise.resolve([]);
}
```